odoo/addons/base/models/ir_mail_server.py

```diff
@@ -668,11 +668,14 @@ def _find_mail_server(self, email_from, mail_servers=None):
         # we have no other choices
         mail_server = mail_servers.filtered(lambda m: not m.from_filter)
         if mail_server:
-            return mail_server[0], email_from
+            return mail_server[0], notifications_email or email_from
```

Comment:
I guess the comment should be updated as well since we should not spoof the from anymore.

Reply:
we don't know, maybe we will

Reply:
Comment could be updated quand même a bit :)